### PR TITLE
Adds network contract system

### PR DESCRIPTION
**Pull request by NataKilar**
## Description of changes
Adds network contracts to the game, a method of managing group membership and regular/hourly payments to and from network accounts. Because contracts can themselves permit other users to grant certain contracts, this allows players to create business hierarchies analogous to those we've had in past iterations. In the future, a system for shares and majority control of contracts may be implemented.

This system is still a work in progress, and requires further testing and documentation

### CODE REVIEW

File: mods/persistence/modules/modular_computers/networking/accounts/account.dm [hunk 1]
```diff
@@ -10,4 +10,57 @@ SAVED_VAR(/datum/computer_file/data/account, outbox)
 SAVED_VAR(/datum/computer_file/data/account, spam)
 SAVED_VAR(/datum/computer_file/data/account, deleted)
 SAVED_VAR(/datum/computer_file/data/account, backup)
-SAVED_VAR(/datum/computer_file/data/account, money_account)
\ No newline at end of file
+SAVED_VAR(/datum/computer_file/data/account, money_account)
+
+/datum/contract_instance/before_save()
```
Comment (PsyCommando) [inline mods/persistence/modules/modular_computers/networking/accounts/account.dm:15]: Can you double check if it's okay not to call the parent proc?

**Review by PsyCommando — DISMISSED** [review]
It looks mostly fine. The UI template is kind of a massive if/else, but other than that it's okay.

File: mods/persistence/modules/modular_computers/networking/accounts/account.dm (hunk 1 above)
Comment (NataKilar) [reply mods/persistence/modules/modular_computers/networking/accounts/account.dm:15]: Fixed. It's probably technically fine, but better safe than sorry.

**Review by PsyCommando — APPROVED** [review]

File: mods/persistence/modules/modular_computers/networking/accounts/account.dm (hunk 1 above)
Comment (PsyCommando) [reply mods/persistence/modules/modular_computers/networking/accounts/account.dm:15]: alright

**Pull request merged**